faunadb-java-dsl/src/main/java/com/faunadb/client/query/Language.java

```diff
@@ -354,1261 +484,2003 @@ public static Expr Value(HighPrecisionTime value) {
     return Fn.apply("insert", ref, "ts", timestamp, "action", action, "params", params);
   }
 
   /**
-   * Creates a new Insert expression.
+   * Inserts a new event in the instance's history.
    *
+   * @param ref the target resource. Type: Reference
+   * @param timestamp the timestamp in which the event will be inserted in the
+   *                  instance's history. Type: Timestamp
+   * @param action the event action
+   * @param params the event's parameters. Type: Object
+   * @return a new {@link Expr} instance
    * @see <a href="https://fauna.com/documentation/queries#write_functions">FaunaDB Write Functions</a>
+   * @see #Ref(Expr, String)
+   * @see #Obj(Map)
+   * @see #Time(Expr)
+   * @see #Value(Instant)
+   * @see #Value(HighPrecisionTime)
    */
   public static Expr Insert(Expr ref, Expr timestamp, Action action, Expr params) {
     return Insert(ref, timestamp, action.value, params);
   }
 
   /**
-   * Creates a new Remove expression.
+   * Removes an event from an instance's history.
    *
+   * @param ref the target resource. Type: Reference
+   * @param timestamp the timestamp in which the event happened. Type: Timestamp
+   * @param action the event's action. Type: event action
+   * @return a new {@link Expr} instance
    * @see <a href="https://fauna.com/documentation/queries#write_functions">FaunaDB Write Functions</a>
+   * @see Action
+   * @see #Ref(Expr, String)
+   * @see #Obj(Map)
+   * @see #Time(Expr)
+   * @see #Value(Instant)
+   * @see #Value(HighPrecisionTime)
    */
   public static Expr Remove(Expr ref, Expr timestamp, Expr action) {
     return Fn.apply("remove", ref, "ts", timestamp, "action", action);
   }
 
   /**
-   * Creates a new Remove expression.
+   * Removes an event from an instance's history.
    *
+   * @param ref the target resource. Type: Reference
+   * @param timestamp the timestamp in which the event happened. Type: Timestamp
+   * @param action the event's action
+   * @return a new {@link Expr} instance
    * @see <a href="https://fauna.com/documentation/queries#write_functions">FaunaDB Write Functions</a>
+   * @see #Ref(Expr, String)
+   * @see #Obj(Map)
+   * @see #Time(Expr)
+   * @see #Value(Instant)
+   * @see #Value(HighPrecisionTime)
    */
   public static Expr Remove(Expr ref, Expr timestamp, Action action) {
     return Remove(ref, timestamp, action.value);
   }
 
   /**
-   * Creates a new Create Class expression.
+   * Creates a new class in the current database.
    *
+   * @param params the class's configuration parameters. Type: Object
+   * @return a new {@link Expr} instance
    * @see <a href="https://fauna.com/documentation/queries#write_functions">FaunaDB Write Functions</a>
+   * @see #Obj(Map)
    */
   public static Expr CreateClass(Expr params) {
     return Fn.apply("create_class", params);
   }
 
   /**
-   * Creates a new Create Database expression.
+   * Creates a new database.
    *
```

Comment:
I really liked the code snippets above, can we add this here.
```
   *  <p>Example:</p>
   * <pre>{@code
   *       client.query(CreateDatabase(Obj("name", Value("my_app")))
   * }</pre>
```

Reply:
I've added the code sample.